apps/web/components/eventtype/EventLimitsTab.tsx

```diff
@@ -457,6 +501,7 @@ export const EventLimitsTab = ({ eventType }: Pick<EventTypeSetupProps, "eventTy
             required
             type="number"
             containerClassName="max-w-80"
+            {...offsetStartLockedProps}
```

Comment:
This was removed recently to allow these fields to be unlocked for v1, now with v2 we need them as the team admin can discretely select their state